db/seeds.rb

```diff
@@ -25,49 +33,9 @@
   end
 end
 
+#-------GLOBAL VARIABLES
 
-#-------METHODS
-
-#Method for prompting the user for the number of records per model they want to seed into the database.
-def ask_for_records(model)
-  formatted_model = model.camelize
-
-  puts "\nHow many #{formatted_model} records would you like to generate? (please enter a number)"
-  STDIN.gets.chomp.to_i
-end
-
-
-def time_rand(from = 0.0, to = Time.now, length = 0, options = {})
-  options[:passes_blackout_validations] = true
-
-  range = to.to_f - from.to_f
-
-  range = length.to_f if length > 0
-
-  random_time = Time.at(from.to_f + rand * range)
-  blackouts = Blackout.all.map { |blk| blk.start_date..blk.end_date }
-
-  if options[:passes_blackout_validations] && !blackouts.blank?
-    while includes?(blackouts, random_time)
-      random_time = Time.at(from.to_f + rand * range)
-    end
-  end
-
-  random_time
-end
-
-def includes?(array_of_ranges, elem)
-  array_of_ranges.each do |rng|
-    if rng.include?(elem)
-      return true
-    else
-      return false
-    end
-  end
-end
-
-def terms_of_service_text
-  %q{
+@terms_of_service_text =  %q{
```

Comment:
Good point. I assumed ruby had no `const` equivalent so I decided to indicate this via global/instance variable use. I just now double checked and I was wrong; ruby does actually have (a kind of weak) `const` so I've changed all instance variables (where appropriate) to const types instead (which are also global)